stl/inc/yvals_core.h

```diff
@@ -1570,9 +1571,9 @@ _EMIT_STL_ERROR(STL1004, "C++98 unexpected() is incompatible with C++23 unexpect
 
 #if defined(__cpp_lib_concepts) // TRANSITION, GH-395
 #if _HAS_CXX23
-#define __cpp_lib_ranges 202202L // P2387R3 Pipe Support For User-Defined Range Adaptors
+#define __cpp_lib_ranges 202207L // P2494R2 Relaxing Range Adaptors To Allow For Move-Only Types
 #elif _HAS_CXX20 // ^^^ _HAS_CXX23 / _HAS_CXX20 vvv
-#define __cpp_lib_ranges 202110L // P2415R2 What Is A `view`?
+#define __cpp_lib_ranges 202110L // P2415R2 What Is A View?

```

Comment:
Mega-mega-nitpick: The long list of papers above mentioned this in lowercase without backticks (since the code-font `view` is meant, but we don't put backticks in these titles).